### PR TITLE
Fix HTTPs -> HTTPS in warning message

### PR DESCRIPTION
**Pull request by CydeWeys**
The entire word is an acronym, so "HTTPS" is correct.  "HTTPs" would be plural HTTP, which isn't the intended use here.

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Fix HTTPs -> HTTPS in warning message  
  [CydeWeys](https://github.com/CydeWeys)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by dnkoutso** [comment]
Needs test update

**Comment by CydeWeys** [comment]
Please feel free to squash commits when merging.

File: CHANGELOG.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   settings for pod targets.  
   [Samuel Giddins](https://github.com/segiddins)
   [#8314](https://github.com/CocoaPods/CocoaPods/pull/8314)
+  
+* Fix HTTPs -> HTTPS in warning message  
+  [CydeWeys](https://github.com/CydeWeys)
+  [#8354](https://github.com/CocoaPods/CocoaPods/pull/8353)
```
Comment (dnkoutso) [inline CHANGELOG.md]: link is wrong: should be `https://github.com/CocoaPods/CocoaPods/issues/8354`

**Comment by CydeWeys** [comment]
Thanks!

**Pull request merged**